lib/senkyoshi/models/questions/matching.rb

```diff
@@ -2,13 +2,25 @@
 
 module Senkyoshi
   class Matching < Question
+    TAGS_PATTERN = Regexp.union(/<\/?p[^>]*>/i,             # <p> tags
```

Comment:
I'm not trying to beat a dead horse here...lol
if you were to multi line it would make it better to add patterns later.

```ruby
TAGS_PATTERN = Regexp.union(
  /<\/?p[^>]*>/i,             # <p> tags
  /<\/?b[^>]*>/i,             # <b> tags
  /<\/?strong[^>]*>/i,        # <strong> tags
  /<\/?em[^>]*>/i,            # <em> tags
  /<\/?span[^>]*>/i,          # <span> tags
  /<\/?i(?!mg)[^>]*>/i,       # <i> tags, no <img>
  / style=("|')[^"']*("|')/i, # inline styles
)
text.gsub(TAGS_PATTERN, "")
```
or
```ruby
TAG_PATTERNS = [
  /<\/?p[^>]*>/i,             # <p> tags
  /<\/?b[^>]*>/i,             # <b> tags
  /<\/?strong[^>]*>/i,        # <strong> tags
  /<\/?em[^>]*>/i,            # <em> tags
  /<\/?span[^>]*>/i,          # <span> tags
  /<\/?i(?!mg)[^>]*>/i,       # <i> tags, no <img>
  / style=("|')[^"']*("|')/i, # inline styles
].freeze

TAGS_RE = Regexp.union(TAG_PATTERNS)
text.gsub(TAGS_RE, "")
```
or
```ruby
TAG_PATTERNS = {
  /<\/?p[^>]*>/i => "",             # <p> tags
  /<\/?b[^>]*>/i => "",             # <b> tags
  /<\/?strong[^>]*>/i => "",        # <strong> tags
  /<\/?em[^>]*>/i => "",            # <em> tags
  /<\/?span[^>]*>/i => "",          # <span> tags
  /<\/?i(?!mg)[^>]*>/i => "",       # <i> tags, no <img>
  / style=("|')[^"']*("|')/i => "", # inline styles
}.freeze

TAGS_RE = Regexp.union(TAG_PATTERNS.keys)
text.gsub(TAGS_RE, TAG_PATTERNS)
```